guides/micronaut-jms-aws-sqs/groovy/src/test/groovy/example/micronaut/MicronautguideTest.groovy

```diff
@@ -0,0 +1,46 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import io.micronaut.test.support.TestPropertyProvider;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers(disabledWithoutDocker = true)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@MicronautTest
+class MicronautguideTest implements TestPropertyProvider {
```

Comment:
why not write the groovy guide tests with Spock instead of Junit5 as we do for every guide?

Reply:
Have replaced it with the Spock test implementation.